potodo/_po_file.py

```diff
@@ -0,0 +1,73 @@
+import polib
+
+from pathlib import Path
+
+
+class PoFile:
```

Comment:
This one looks a lots of lines of code for a few things. Subclassing the weird polib is not easy neither, maybe try:

```python
 class PoFile:
    def __init__(self, path: Path):
        self.path: Path = path
        self.pofile: polib.POFile = polib.pofile(path)


    def __str__(self) -> str:
        return (
            f"Filename: {self.path.name}\n"
            f"Fuzzy Entries: {self.pofile.fuzzy_entries()}\n"
            f"Percent Translated: {self.pofile.percent_translated()}\n"
            f"Translated Entries: {self.pofile.translated_entries()}\n"
            f"Untranslated Entries: {self.pofile.untranslated_entries()}"
        )
```

I won't even implement __lt__ (I'd use the key parameter of sorted, like `sorted(..., key=lambda po: po.path.name)`)

Reply:
Hum, I dropped some attributs you use elsewhere, like `translated_nb` and so on.

Reply:
So maybe just ignore this comment, but maybe rename the class `POFileStats`.